src/main/resources/public/jsx/Dataset.jsx

```diff
@@ -241,6 +250,8 @@ class Dataset extends React.Component {
                                          licenses={this.state.licenses}
                                          license={this.state.fields['license']}
                                          tags={this.state.fields.tags}
+                                         organizations={this.state.organizations}
+                                         organization={this.state.fields['organization']}
```

Comment:
As said above, do not hesitate to be precise in the content of the variable (is it organization id ? name ? ...)